packages/hurl/src/report/junit/testcase.rs

```diff
@@ -132,6 +132,7 @@ HTTP/1.0 200
                 )],
                 time_in_ms: 0,
                 compressed: false,
+                max_filesize: 0,
```

Comment:
This code is not necessary now, it can't compile (you can test it with `cargo test --lib`)